app/scripts/modules/kubernetes/src/pipeline/stages/runJob/runJobStage.js

```diff
@@ -62,10 +49,36 @@ module.exports = angular.module('spinnaker.kubernetes.pipeline.stage.runJobStage
       this.stage.dnsPolicy = 'ClusterFirst';
     }
 
-    if (this.stage.container.imageDescription.fromTrigger === true) {
```

Comment:
What makes this unnecessary now?

Reply:
`fromTrigger` is now set when the image is selected. It's populated as part of the `searchImages` function.